test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/FilterAndOrderByFunctionalTests.cs

```diff
@@ -2375,6 +2385,118 @@ public void FilterWithInOperationWithEmptyCollection(string filterClause)
             Assert.Equal(0, collectionNode.Collection.Count);
         }
 
+        [Theory]
+        [InlineData("SSN in ('')")]     // Edm.String
+        [InlineData("SSN in ( '' )")]     // Edm.String
+        [InlineData("SSN in (\"\")")]     // Edm.String
+        [InlineData("SSN in ( \"\" )")]     // Edm.String
+        public void FilterWithInOperationWithEmptyString(string filterClause)
+        {
+            FilterClause filter = ParseFilter(filterClause, HardCodedTestModel.TestModel, HardCodedTestModel.GetPersonType());
+
+            var inNode = Assert.IsType<InNode>(filter.Expression);
+
+            CollectionConstantNode collectionNode = Assert.IsType<CollectionConstantNode>(inNode.Right);
+            Assert.Equal(1, collectionNode.Collection.Count);

```

Comment:
We don't need to assert that the element of the collection is actually an empty string? It's possible the implementation may contain bugs and we appended something else to the collection :)

Reply:
#Resolved